models/contacts.js

```diff
@@ -1,20 +1,20 @@
 const { readFile, writeFile } = require("fs/promises");
+const Joi = require("joi");
 const { v4: uuidv4 } = require("uuid");
 const path = require("path");
 require("colors");
 
-const baseDir = __dirname;
 const fileName = "contacts.json";
-const contactsPath = path.join(baseDir, fileName);
+const contactsPath = path.join(__dirname, fileName);
 
-const contactsArray = () => {
-  return readFile(contactsPath, "utf-8")
+const getContactsArray = async () => {
+  return await readFile(contactsPath, "utf-8")
     .then((data) => JSON.parse(data))
     .catch((error) => console.log(error.message.red));
 };
 
-const saveContactArray = (contacts) => {
-  writeFile(contactsPath, JSON.stringify(contacts, null, 2), (error) => {
+const saveContactsArray = async (contacts) => {
+  await writeFile(contactsPath, JSON.stringify(contacts, null, 2), (error) => {
```

Comment:
```suggestion
  await writeFile(contactsPath, JSON.stringify(contacts, null, 2));
```
To jest poprawna wersja